src/components/existing-request/existing-request-display.vue

```diff
@@ -734,6 +775,117 @@ export default class ExistingRequestDisplay extends Mixins(
       this.$root.$emit('showSpinner', false)
     }
   }
+
+  private get isThereMoreThanOnePaymentMethod () {
```

Comment:
This method and the ones below could probably benefit from return types and methods comments/doc block comments.

Reply:
👍